web/src/utils/filter-definitions.ts

```diff
@@ -448,6 +449,34 @@ export const getFilterDefinitions = (
         encoder: simpleFiltersEncoder('_HashId'),
         overlap: false
       },
+      {
+        id: 'pkt_drop_state',
```

Comment:
well this filter be off when the dropped packet is for UDP as an example ?

Reply:
No specific mechanism is done here. The user can set 2 filters as `PktDropLatestState=TCP_XXX`, `Proto=UDP` with `match any` query option to get both dropped TCP & UDP flows